### ISSUE TITLE
Изображения имеют большой размер

### ISSUE THREAD

**Issue by RusPosevkin** [RusPosevkin/evolution#4]
Уменьшить размер изображений.
PNG по 1.4 и 1.8 Mb  нам не нужны :)

**Comment by krauklish** [comment]
Да, надо оптимизировать. Пока даже не пытался, выгрузил как есть.

**Comment by RusPosevkin** [comment]
@krauklish 
В https://github.com/RusPosevkin/evolution/commit/443436ed7ecd5987cdb901e6cc496529ca9d606e ты все изображения заоптимизировал? Или могут быть еще исправления?

**Comment by krauklish** [comment]
@RusPosevkin тут был вопрос? Уведомление есть, а коммент не вижу :)

В любом случае пишу, чтоб не забыть. Надо переделать:
- africa-kenya и africa-chad (поменился стиль);
- ardi-skeleton (нет hd, надо отделить тело от скелета);
- hand-chopper (нет hd);
- homo-heidelbergensis (нет hd, надо отредактировать);
- scheme-star (поменяется стиль);
- journey-map-shell (вроде нет hd, но может и не надо);
- все timeline- картинки.

Сейчас я все загружаю @2x, чтобы поддерживать ретину и не запариваться с двойной выдачей. «Нет hd» значит, что пока картинка @1x. Соответственно, для всех неадаптивных картинок нужно прописывать разрешение.

**Comment by RusPosevkin** [comment]
Да, я сначала задал вопрос, а потом сделал `git pull`, посмотрел на картинки, и понял, что есть еще над чем поработать :)
